libs/plugins/stream-client/src/lib/core/state/clean-dangling-tasks-output-mappings.ts

```diff
@@ -1,43 +1,40 @@
 import { isEmpty, pick, fromPairs } from 'lodash';
-import { ContributionSchema, isMapperActivity } from '@flogo-web/core';
 import { Dictionary } from '@flogo-web/lib-client/core';
 
 import { Item } from '../interfaces';
 import { FlogoStreamState } from './stream.state';
+import { ROOT_TYPES } from '../../shared/mapper/constants';
 
 /**
  * When stream schema's output change we need to remove the task mappings that were referencing them
  * @param state
  */
 export function cleanDanglingTaskOutputMappings(state: FlogoStreamState) {
-  const outputNames =
+  let outputNames =
     state.metadata && state.metadata.output
       ? state.metadata.output.map(o => o.name)
       : null;
   if (!outputNames) {
     return state;
   }
 
-  const cleanItems = itemCleaner(state.schemas, outputNames);
+  outputNames = normalizeOutputs(outputNames);
```

Comment:
nitpick but I think this should be called something other than `normalize` since it is not "normalizing" the output names. I can't think of anything good though, `prependPipelineResolver()` feels too verbose...

edit: what if instead we do:

```typescript
outputNames = outputNames.map(prependPipeline);
```

where `prependPipeline` is just what we have in the current normalize function, this:

```
function prependPipeline(outputName: string) {
	return `$${ROOT_TYPES.PIPELINE}.${outputName}`;
}
```